cspell.json

```diff
@@ -30,11 +30,5 @@
         "**/*.trie",
         "**/.vscode/**"
     ],
-    "ignoreWords": [
```

Comment:
Ignoring `cspell.json` files might also be useful since they are also automatically updated by VS Code editor.

Reply:
Excluded the `.vscode` folder